### ISSUE TITLE
SendCloudApiParcelDocumentsResource.Download - Double slash in URL results in 404 response

### ISSUE THREAD

**Issue by lukas-iamd** [janssenr/SendCloudApi.Net#24]
Hi

Downloading Parcel Documents doesn't work anymore.

The slash after the HostUrl must be removed as the host URL ends with a slash. The SendCloud API does not accept this anymore and returns a 404 Not Found error.
```csharp
// File SendCloudApiParcelDocumentsResource.cs
// Line 21
 Uri uri = Client.GetUrl($"{HostUrl}/parcels/{parcelId}/documents/{type}", parameters);
```

**Comment by janssenr** [comment]
Removed the slash after the HostUrl.

**Comment by lukas-iamd** [comment]
Hi

Thanks for the fast response. When could we expect a new release to be published with this fix?

**Comment by janssenr** [comment]
I just published release 2.0.25 to NuGet. This release also includes support for V3 of the SendCloud API, so there might be some more changes you have to make.
